src/theme/srTheme/colors.js

```diff
@@ -4,6 +4,7 @@ const colors = {
   blue: '#223c59',
   blue_dark: '#3a4972',
   blue_donate: '#2042AD',
+  blue_donate_interact: '#001A85',
```

Comment:
Probably only really need this in the crTheme, but I guess there's no harm. 

Not even sure how much this Sport Relief theme gets used nowdays